.github/CODEOWNERS

```diff
@@ -1 +1,2 @@
-app/tests @jrbenny35 @pdehaan
\ No newline at end of file
+app/tests @jrbenny35 @pdehaan
+app/experimenter/base/migrations @jaredlockhart @lmorchard
```

Comment:
@lmorchard What about the migrations at [app/experimenter/experiments/migrations](https://github.com/mozilla/experimenter/tree/main/app/experimenter/experiments/migrations)?

Reply:
Good catch! I missed 'em!

I just ran `find -type d -name 'migrations'` and added everything I found, although I don't know what some of these are for

Reply:
Oh, you found way more folders. Well, now I'm wondering if they support complex globs and we could do something like `app/experimenter/**/migrations`.

The docs don't explicitly show double star glob in https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax, but they do say they follow the .gitignore syntax (with a few exceptions).